snmp/fail2ban

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env perl
+
+# Add this to your snmpd.conf as below.
+# extend fail2ban /etc/snmp/fail2ban
+#
+# Then add to your cron tab, if you wish to use caching.
+# */3 * * * * /root/snmp-extends/fail2ban.pl -u
+
+#make sure this path is correct
+my $f2bc="/usr/local/bin/fail2ban-client";
```

Comment:
This is still going to be incorrect on most systems.
My suggestions:
`fail2ban-client`
`/usr/bin/env fail2ban-client`